libandroid-navigation-ui/src/main/java/com/mapbox/services/android/navigation/ui/v5/instruction/InstructionView.java

```diff
@@ -156,8 +159,11 @@ public void onFeedbackDismissed() {
    * @param navigationViewModel to which this View is subscribing
    * @since 0.6.2
    */
-  public void subscribe(NavigationViewModel navigationViewModel) {
+  public void subscribe(NavigationViewModel navigationViewModel,
+                        Locale locale, @NavigationUnitType.UnitType int unitType) {
```

Comment:
Passing locale and unitType here will be removed / replaced with setter methods like we discussed